src/System.IO.Compression.Brotli/System/IO/Compression/Brotli/Brotli.cs

```diff
@@ -36,10 +42,14 @@ private static TransformationStatus GetTransformationStatusFromBrotliDecoderResu
             if (result == BrotliDecoderResult.NeedsMoreInput) return TransformationStatus.NeedMoreSourceData;
             return TransformationStatus.InvalidData;
         }
+        public static TransformationStatus Compress(ReadOnlySpan<byte> source, Span<byte> destination, out int bytesConsumed, out int bytesWritten, CompressionLevel quality = (CompressionLevel)DefaultQuality, int windowSize = DefaultWindowSize, BrotliEncoderMode encMode = BrotliEncoderMode.Generic)
```

Comment:
I am curios, why is the DefaultQuality 11 and not one of the predefined CompressionLevel enum values?